README.md

```diff
@@ -30,14 +32,31 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+Map creates a new array and does not manipulate the original array. For each goes over every item in the array and you can manipulate it however you like with a callback.
+
 2. What is the difference between a function and a method?
 
+A method is a function that belongs to a class.
+
 3. What is closure?
 
+A function that creates its own scope. It gives access to outer function scopes as well.
+
```

Comment:
currently accessible environment at any point in a program. created by different nested scopes which are created by various bracket pairs :)